### PR TITLE
Coding intro - PQs

### PR DESCRIPTION
**Pull request by abdullahtahir147**

### CODE REVIEW

File: coding-intro/coding-intro-core/core-concepts/do-it-yourself.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ setupCode:
 ---
 ## Content
 
-Welcome to your first coding task.
+Let's write a Python program!
```
[the remark this answers is not in the document]
Comment (abdullahtahir147) [reply coding-intro/coding-intro-core/core-concepts/do-it-yourself.md:23]: How is it now?

[the remark this answers is not in the document]
Comment (Stefan-Stojanovic) [reply coding-intro/coding-intro-core/core-concepts/do-it-yourself.md:23]: Maybe like this:
```
setupCode:
  startingPoint: |
    # Who did you meet today?
    # Note: One of the print statements has an error!
    name = "???"
    age = ???
  
    print(age) # Print age to the screen
    print(name/2) # Print name to the screen
    
```

[the remark this answers is not in the document]
Comment (Stefan-Stojanovic) [reply coding-intro/coding-intro-core/core-concepts/do-it-yourself.md:23]: This way, they still have to figure out which one has an error. What do you think?

[the remark this answers is not in the document]
Comment (abdullahtahir147) [reply coding-intro/coding-intro-core/core-concepts/do-it-yourself.md:23]: Perfect

**Review by Stefan-Stojanovic — APPROVED** [review]

**Pull request merged**